types/backbone/index.d.ts

```diff
@@ -267,7 +267,7 @@ declare namespace Backbone {
          *    return super.get("name");
          * }
          */
-        get<a extends keyof T & string>(attributeName: a): T[a];
+        get<A extends _StringKey<T>>(attributeName: A): T[A] | undefined;
```

Comment:
Why add `undefined`?

Reply:
If you didn't set attribute to model, you can get undefined https://github.com/jashkenas/backbone/blob/master/backbone.js#L451

Reply:
Because it might return `undefined` and some people enable `strictNullChecks`.

Reply:
And that's not captured in the type `T[A]`?

Reply:
Yep

Reply:
Fair enough.